lib/matplotlib/tests/test_axes.py

```diff
@@ -2932,7 +2932,7 @@ def test_hist_stacked_normed():
     d2 = np.linspace(0, 10, 50)
     fig = plt.figure()
     ax = fig.add_subplot(111)
-    ax.hist((d1, d2), stacked=True, normed=True)
```

Comment:
I think this should stay as it is, because we should still be testing that `normed` works until it is removed.

Reply:
In that case, shouldn't the test have two forms, one for 'normed' and the other for 'density'?  At a more general level, we never have tested everything, and never will--there are too many possible combinations.  We need to concentrate on the most useful tests, putting our resources, and those of the CI services, where they will be most effective in improving mpl.  Testing that a deprecated kwarg still works seems very low priority, doesn't it?  The time of deprecation seems like a perfectly reasonable time to switch a test to the non-deprecated alternative.

Reply:
- There is a test directly below this one for `density`.
- Sure we've never tested everything, but we should *aim* to test everything. This test definitely isn't a CI bottleneck.
- In my opinion, something being deprecated means it is still supported, but there are plans for it to be removed. If it's supported, it should be tested.

I'm not going to block on removing the `normed` test, but I am -2 on it.

Reply:
Aiming to test everything doesn't make sense to me, for the reason stated above.